### PR TITLE
(WIP) Create a github release

### PR DESCRIPTION
**Pull request by vinzent**
Additionlly to the puppet forge push also create a release
on github.

* needs an encrypted github access token 
* don't know what it writes as text or as title

### CODE REVIEW

File: moduleroot/.travis.yml.erb [hunk 1]
```diff
@@ -76,13 +76,22 @@ branches:
...
+      secure: "<%= @configs['secure'] -%>"
+    on:
+      tags: true
+      # all_branches is required to use tags
+      all_branches: true
+      # Only publish the build marked with "DEPLOY_TO_FORGE"
+      condition: "$DEPLOY_TO_FORGE = yes"
+  - provider: releases
+    api_key: 
+      secure: "GITHUB OAUTH TOKEN"
```
Comment (bastelfreak) [inline moduleroot/.travis.yml.erb]: I'm not sure who can generate this, probably only people with admin terms? We should move the generation of the token to plumbing and read it here from the .sync.yml (as we do it with the string in line 82)

Comment (rnelson0) [inline moduleroot/.travis.yml.erb]: I think since this would be a org oauth token, it would be the same for all modules.

Comment (vinzent) [inline moduleroot/.travis.yml.erb]: is the travis encryption of the key by repo or by org?

**Comment by ghoneycutt** [comment]
@vinzent I don't get the point in this as when you submit a tag to github, then github creates a release. This code would only create releases when a tag exists. If a tag exists then wouldn't a github release also already exist?

**Comment by ghoneycutt** [comment]
Closing due to lack of activity, please feel free to re-open.

**Pull request closed without merging**